src/chrome/content/rules/EtaLabs.net.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="EtaLabs.net">
+	<target host="etalabs.net" />
+	<target host="www.etalabs.net" />
+	<target host="git.etalabs.net" />
+
+	<rule from="^http://etalabs\.net/" to="https://www.etalabs.net/" />
```

Comment:
Please [document](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts) the issue on `^`.